sklearn/_build_utils/openmp_helpers.py

```diff
@@ -40,6 +40,8 @@ def get_openmp_flag(compiler):
         # export LDFLAGS="$LDFLAGS -Wl,-rpath,/usr/local/opt/libomp/lib
         #                          -L/usr/local/opt/libomp/lib -lomp"
         return []
+    elif sys.platform == "linux" and "icc" in compiler:
```

Comment:
I think this applies to `sys.platform == "darwin"` as well.

Reply:
For darwin we set `-openmp`. Is it outdated or preferable to use `-qopenmp` ?

Reply:
also is `icl` something ? I remember putting `icc or icl` because I had the feeling that it could be a name of the icc executable but I actually never see that anywhere.

Reply:
Yes, it is `icl` on Windows.